src/chain.js

```diff
@@ -11,19 +11,25 @@ var map = require('./map');
  *
  * Dispatches to the `chain` method of the second argument, if present,
  * according to the [FantasyLand Chain spec](https://github.com/fantasyland/fantasy-land#chain).
+ * Treats the second argument as a FantasyLand Chain, when the the second argument is a function
+ * of the form `a -> x -> b`.
```

Comment:
Not in this case. Did you see the example? `chain(head, append)`. Surely, the second argument is the `Chain` (`a -> x -> b`) here, and we are using dispatchiness to dictate how the arguments are consumed?